src/components/ValueSet/ValueSetForm.tsx

```diff
@@ -35,8 +35,16 @@ import valuesetApi from "@/types/valueset/valuesetApi";
 
 // Create a schema for form validation
 const valuesetFormSchema = z.object({
-  name: z.string().min(1, "Name is required"),
-  slug: z.string().min(1, "Slug is required"),
+  name: z.string().trim().min(1, "Name is required"),
```

Comment:
let's switch to using field_required translation key (i.e. `t("field_required")`).